client/src/components/AdminPageUsers.jsx

```diff
@@ -0,0 +1,156 @@
+import React, { Component } from 'react';
+import { Table, Input } from 'react-materialize';
+import { request } from '../utils';
+
+
+class AdminPageUsers extends Component {
+  state = {
+    users: [],
+    roles: [],
+    statuses: []
+  };
+
+  componentDidMount() {
+    request('/admin-page/users')
+      .then(data => {
+        this.setState({ users: data.users_dict });
+        this.setState({ roles: data.roles_dict})
+        this.setState({ statuses: data.statuses_dict})
+      })
+  }
+
+  changeUserRole = (e) => {
+    let { id } = e.currentTarget;
+    id = id.slice(10);
```

Comment:
It's can be hard to understand what `10` means, you can use here `id = id.replace(/user-role-/g, '')`

Reply:
What is "g"?